run_test.go

```diff
@@ -132,6 +132,30 @@ func TestRunsCreate(t *testing.T) {
 		assert.Equal(t, true, r.RefreshOnly)
 	})
 
+	t.Run("with auto-apply requeted", func(t *testing.T) {
+		// ensure the worksapce auto-apply is false so it does not default to that.
+		assert.Equal(t, false, wTest.AutoApply)
+
+		options := RunCreateOptions{
+			Workspace: wTest,
+			AutoApply: Bool(true),
+		}
+
+		r, err := client.Runs.Create(ctx, options)
+		require.NoError(t, err)
+		assert.Equal(t, true, r.AutoApply)
```

Comment:
Hmm. This seems to fail.

Reply:
I think there is an issue in the tfe instance. This test passes locally, staging, and production. I am re-deploying the test instance now.